WooCommerce/Classes/Tools/RequirementsChecker.swift

```diff
@@ -100,16 +100,7 @@ private extension RequirementsChecker {
         guard let baseViewController else {
             return
         }
-        UIAlertController.presentExpiredWPComPlanAlert(from: baseViewController) {
-            if site.wasEcommerceTrial {
-                /// If site once ran a trial WooExpress plan, attempt show IAP if possible.
-                UpgradesViewPresentationCoordinator().presentUpgrades(for: site.siteID, from: baseViewController)
-            } else {
-                /// If the site never ran a WooExpress plan, show the appropriate plans on the web.
-                let controller = UpgradePlanCoordinatingController(siteID: site.siteID, source: .expiredWPComPlanAlert)
-                baseViewController.present(controller, animated: true)
-            }
-        }
+        UIAlertController.presentExpiredWPComPlanAlert(from: baseViewController)
```

Comment:
This is being called by `checkEligibilityForDefaultStore()` above (line 63) and it seems to deal with checking for default store case (different from switching to an expired site case that is handled in `StorePickerViewModel`).

In this case, `UIAlertController.presentExpiredWPComPlanAlert` still have functionality and old text for upgrading. I think it should be updated as well?

Reply:
Thanks for the review, @hafizrahman! Sorry, I don't understand. 

> and it seems to deal with checking for default store case

I have removed the callback in https://github.com/woocommerce/woocommerce-ios/pull/12406/files#diff-069f9c0e77b4fbdec872c56a9dae6c6a5039ded25ce788db8a780a716ca51b6e. 

> In this case, UIAlertController.presentExpiredWPComPlanAlert still have functionality and old text for upgrading. I think it should be updated as well?

We should not see the old text anymore as https://github.com/woocommerce/woocommerce-ios/pull/12406/files#diff-2b823319b589a610c38dfec95983e11e3668889367a5ea183571473fd8d657be is updated with new text and both ``RequirementsChecker` and `StorePickerViewController` use the helper method from `UIAlertController+Helpers.swift` for presenting the alert. 

Kindly let me know if you are referring to something else.

Reply:
Oh! I was confused because I was looking at a different branch in my local repo 🤦🏼 

Yup, this looks good now, sorry for confusing you.

Reply:
No problem. Thank you!